lab/lab1/part4-project-euler.html

```diff
@@ -31,7 +31,40 @@
     Start code
 
     ===================== */
-
+//Problem #1
+//If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.
+//Find the sum of all the multiples of 3 or 5 below 1000.
+//Worked with Ian Schwarzenberg
+//Code support found via: https://codereview.stackexchange.com/questions/77018/project-euler-1-javascript
+var emptyArray = [];//creates an empty array
+var answer = 0;//establishes sum variable for use in loop
+
+for (var i = 1; x < 1000; x++) {//establishes look from 1 to 1000
```

Comment:
you switch between x and i... making it not actually load.

![image](https://user-images.githubusercontent.com/8103418/52543748-f9fe5c00-2d79-11e9-882a-692d5d9e2bcb.png)